### DISCUSSION TITLE
this is a server webhook without a client

### DISCUSSION THREAD

**Discussion by Baz2024uk** [chadwpalm/Lumunarr#28] [Q&A] [answered]
have this running in docker on my sysnology and works really well have noticed in the logs in docker this and not sure what it means can you advise please 
(INFO)this is a server webhook without a client

**Comment by chadwpalm** [comment, marked as answer]
Not all webhooks that are sent from Plex Media Servers are associated with Plex clients. There are some related to alerts among other events, but since Lumanarr receives all webhooks and parses them out, if it comes across a webhooks that wasn't triggered by a Plex client that is the output it gives.

It's not an error message, just informational.

**Reply by Baz2024uk** [reply]
Thank you for explaining,  you have done a great job with this love how it works so well
